assets/sass/okta-theme.scss

```diff
@@ -81,6 +81,8 @@
 }
 
 #okta-sign-in.auth-container {
+  // for IE to support main tag's width
+  display: block;
```

Comment:
are we removing `main` ?

Reply:
Adding display: block so IE recognize's main tag's width.

Reply:
should we also add that this change is only for IE in CSS ? like make it a browser specific change instead of a general css change.

Reply:
I just saw this comment

Reply:
`display` does not have browser vendor specific prefixes, so how would one do that in this codebase?

Reply:
there is way to do it, sending you SO link. Now this is closed, we can do it later (probably not a priority)